website/blog/2020-05-25-7.10.0.md

```diff
@@ -0,0 +1,128 @@
+
+<!-- truncate -->
+
+Alongside this Babel release, we are releasing the first experimental version of our new polyfills compatibility architecture. We began discussions about this over a year ago in a [RFC issue](https://github.com/babel/babel/issues/10008) within the Babel repository. As an aside, we now have an official RFC process for discussing changes that significantly impact our users: you can check it out over in the [`babel/rfcs`](https://github.com/babel/rfcs) repository!
+
+Also if you have any questions, we've enabled [GitHub Discussions](https://github.com/babel/babel/discussions) on our repository!
+
+If you or your company want to support Babel and the evolution of JavaScript, but aren't sure how, you can donate to us on our [Open Collective](https://opencollective.com/babel) and, better yet, work with us on the implementation of [new ECMAScript proposals](https://github.com/babel/proposals) directly! As a volunteer-driven project, we rely on the community's support to fund our efforts in supporting the wide range of JavaScript users. Reach out at [team@babeljs.io](mailto:team@babeljs.io) if you'd like to discuss more!
+
+## Highlights
+
+### New features enabled by default
+
+Now that it has reached stage 4, parsing for `import.meta` is enabled by default. Please note that `@babel/preset-env` doesn't support transforming it, because what that object contains is up to the engine and is not defined in the ECMAScript specification.
+
+We also discovered that we still didn't support compiling a 5-years-old ECMAScript feature: `\u{...}`-style Unicode escapes! Thanks to [Justin](https://github.com/jridgewell), `@babel/preset-env` can now compile them by default both in strings and in identifiers.
+
+Lastly, we have added `@babel/plugin-proposal-class-properties` and `@babel/plugin-proposal-private-methods` to the [`shippedProposals`](https://babeljs.io/docs/en/babel-preset-env#shippedproposals) option of `@babel/preset-env`. These proposals are not Stage 4 (i.e. part of the ECMAScript standard) yet, but they are already enabled by default in [many JavaScript engines](https://github.com/tc39/proposal-class-fields#implementations).
+
+### Improved optional chaining `?.` ergonomics ([#10961](https://github.com/babel/babel/pull/10961), [#11248](https://github.com/babel/babel/pull/11248))
+
+In TypeScript 3.9, the interaction between non-null assertions (postfix `!`) and optional chaining [has been changed](https://www.typescriptlang.org/docs/handbook/release-notes/typescript-3-9.html#parsing-differences-in-optional-chaining-and-non-null-assertions), to make it more useful.
+
+```ts
+foo?.bar!.baz
+```
+
+In TypeScript 3.8 and Babel 7.9, that could would be read as `(foo?.bar)!.baz`: "If `foo` is not nullish, get `.bar` from it. Then trust me that `foo?.bar` is never nullish and always get `.bar` from it". This means that when `foo` is nullish that code would always throw, because we are trying to get `.baz` from `undefined`.
```

Comment:
```suggestion
In TypeScript 3.8 and Babel 7.9, the above would be read as `(foo?.bar)!.baz`: "If `foo` is not nullish, get `.bar` from it. Then trust me that `foo?.bar` is never nullish and always get `.bar` from it". This means that when `foo` is nullish that code would always throw, because we are trying to get `.baz` from `undefined`.
```